### ISSUE TITLE
Videos not playing after Compiling in Air SDK 50.0.1.2

### ISSUE THREAD

**Issue by GithubProgrammer** [airsdk/Adobe-Runtime-Support#2333]
Hello,

I have few years old Desktop Air project that plays audios and videos locally. I have used the code with new Air SDK 50.0.1.2 for adding some features. While testing, it works fine. But after compiling the videos do not play at all. It is almost impossible to dig into the code as it is really huge. So, I have used simple few lines of code to play the sample video. And this also works fine with testing but after compiling, it does not. Here is the code,

import flash.net.NetConnection;
import flash.net.NetStream;
import flash.media.Video;

var nc:NetConnection = new NetConnection();
nc.connect(null);

var ns:NetStream = new NetStream(nc);
ns.client = {};
ns.client.onMetaData = ns_onMetaData;

var myVideo:Video = new Video();
addChild(myVideo);
myVideo.attachNetStream(ns);

ns.play("/Users/mac/sia.mp4");

function ns_onMetaData(item:Object):void {
    trace("metaData");

Thanks.

**Comment by ajwfrost** [comment]
My assumption with that is that you're using a reasonably recent version of macOS and are therefore hitting a permission error when you try to access the `/Users/mac/sia.mp4` file. Can you put it into `Documents`?

**Comment by GithubProgrammer** [comment]
Hi ajwfrost,

Thanks for your quick reply.
I am using Catalina (10.15.6) and Animate 21.0. The same code works fine with AIR SDK 32 (that is the only one I have beside 50.0.1.2)

I have also tested by putting the file into Documents as you have suggested. Same result. The path used above was just a dummy one. I have even tried by browsing the file within the application, but the result is same. Looks like this is not permission related error rather, a new AIR SDK related.

Thanks

**Comment by ajwfrost** [comment]
Thanks -> and Catalina doesn't have those restrictions, I think it was only Monterey that we noticed those other permission problems...

Weird, we can check this though. But:
>  While testing, it works fine. But after compiling the videos do not play at all.

So from Animate, when you do "debug movie", it works .. but "after compiling", what do you mean? Is this when you're creating a bundle via the Publish option? and then launching it via Finder?

I'm still wondering if there's something about sandboxing that may be interfering, but it would be good if we could try to reproduce the exact thing you're doing there..

**Comment by GithubProgrammer** [comment]
Yes, when I test by clicking Command + Enter (debug movie), it works fine. But after using the publish option (with runtime Embedded), it does not work. 

Thanks

**Comment by gravity-ac** [comment]
Did you find a solution for this?

I had an app published that stopped playing video after working flawlessly for years. My particular issue tuned out to be NetStream events. Some of them simply stopped working. The video would load but would not begin playing.

https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/net/NetStream.html

Sorry for the vague answer, but if you are still looking for a fix, this might be one avenue of inquiry to consider.

**Comment by GithubProgrammer** [comment]
No, I have not heard anything on this issue from the Harmans team. I am still using the same old Adobe Air SDK, there is no point on using Harmans SDK without the issue getting resolved. I hope there will some quick response from the team.
